components/compositing/windowing.rs

```diff
@@ -101,8 +101,8 @@ pub enum EmbedderEvent {
     /// Sent when the user triggers a media action through the UA exposed media UI
     /// (play, pause, seek, etc.).
     MediaSessionAction(MediaSessionActionType),
-    /// Set browser visibility. A hidden browser will not tick the animations.
-    ChangeBrowserVisibility(TopLevelBrowsingContextId, bool),
+    /// The visibility of the native window has changed.
+    WindowVisibility(bool),
```

Comment:
I'm curious why this now is a property of the window, rather than the "Browser". Theoretically, an application that supports tabs may want to adjust the visibility of each WebView individually.

Reply:
iirc the idea was that this would make things easier for the embedder under this intermediate model of multiple webviews but only one window, even though we would probably need to change it once we do multiple windows. but letting the embedder handle it would work too, while hugely reducing complexity.

Reply:
fixed in 9e307cfcefed26ecb65399ce12a7e2a7ef38e5d7